src/cmd-buildextend-aws

```diff
@@ -14,16 +14,21 @@ from cmdlib import run_verbose, write_json, sha256sum_file, Builds
 # Parse args and dispatch
 parser = argparse.ArgumentParser()
 parser.add_argument("--build", help="Build ID")
-parser.add_argument("--region", help="EC2 region",
-                    required=True)
-parser.add_argument("--bucket", help="S3 Bucket",
-                    required=True)
+parser.add_argument("--region", help="EC2 region")
+parser.add_argument("--bucket", help="S3 Bucket")
 parser.add_argument("--name-suffix", help="Suffix for name")
 parser.add_argument("--grant-user", help="Grant user launch permission",
                     nargs="*", default=[])
 parser.add_argument("--log-level", help="ore log level")
+parser.add_argument("--upload", action='store_true', default=False,
+                    help="Do or don't upload to AWS")
```

Comment:
I think just `"Upload to AWS"` as the help string would be easier to understand.

Reply:
👍 - will update